internal/parser/lexer.go

```diff
@@ -676,9 +673,9 @@ func (l *lexer) lexSymbol() error {
 				}
 			}
 			if allStar {
-				for _, l := range lines {
-					if l[0] == '*' {
-						l = " " + l
+				for i := range lines {
```

Comment:
@sparkprime I don't really understand the point of this. Why do we need this "little hack"?

FYI before my change this whole loop was a noop, because `l` was a copy.